ExampleMod/Content/Items/Weapons/HitModifiersShowcase.cs

```diff
@@ -25,6 +26,33 @@ public class HitModifiersShowcase : ModItem
 		private const int numberOfModes = 8;
 		private int mode = 0;
 
+		public const int ExtraKnockback = 50;
+		public const int ExtraCriticalHitDamage = 200;
+		public const int ExtraArmorPenetration = 10;
+		public const int ExtraScalingArmorPenetration = 50;
+
+		public static LocalizedText SwitchingText { get; private set; }
+
+		public static LocalizedText CurrentModeText { get; private set; }
+
+		public static LocalizedText[] ModeText { get; private set; }
+
+		public static LocalizedText UnknownModeText { get; private set; }
+
+		public override void SetStaticDefaults() {
+			SwitchingText = this.GetLocalization("Switching");
+			CurrentModeText = this.GetLocalization("CurrentMode");
+			ModeText = [this.GetLocalization("Mode_0"),
```

Comment:
`Enumerable.Range(0, 8).Select(i => this.GetLocalization($"Mode_{i}")).ToArray();`